py/client-ticking/tests/test_ticking_lists.py

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+#
+
+import unittest
+
+import pyarrow as pa
+import pydeephaven as dh
+import datetime as dt
+import queue
+from collections import defaultdict
+from typing import List
+
+class TickingListsTestCase(unittest.TestCase):
+    queue = queue.Queue()
+    errors: List[str] = []
+
+    def test_ticking_lists(self):
+        session = dh.Session()
+        table = session.empty_table(size = 10).update(
```

Comment:
As I read it this table will not actually tick, do we need to do something that would force it to be refreshing (like joining a time table onto it), or are we sufficiently happy with the test against a static table?

Reply:
We're still subscribed to it over Barrage, so I think of this static table as being a ticking table that "ticks once" for the initial snapshot and then doesn't tick any more.

I've got other tests where the table actually ticks multiple times. Personally I like each test to be sensitive to one thing, and in this case the purpose of the test is to see if list types successfully make it to the client over Barrage.

In my opinion making it actually tick multiple times means a more complicated test that takes longer to run and I've already got a test for that.